lib/Backend/Lower.cpp

```diff
@@ -10920,11 +10920,12 @@ Lowerer::LowerArgInAsmJs(IR::Instr * instrArgIn)
 {
     Assert(m_func->GetJITFunctionBody()->IsAsmJsMode());
 
-    int argCount = (int)(int16)m_func->argInsCount;
+    Js::ArgSlot argCount = m_func->argInsCount;
     IR::Instr * instr = instrArgIn;
 
-    for (int argNum = argCount - 1; argNum >= 0; --argNum)
+    for (Js::ArgSlot argNum = 0; argNum < argCount; ++argNum)
```

Comment:
I don't think we need this loop at all. We can let the main lowering loop go one by one. This seems superfluous.